### PR TITLE
Reimplement chapter_leader page

### LINKED ISSUE #167: Reimplement /chapter_leader page
# Feature
## Why is this feature being added?

https://operationcode.org/chapter_leader exists and it needs to exist in the new code base as well.

## What should your feature do?
- Implement the chapter_leader page via `pages/chapter_leader.js`
- [See live source code](https://github.com/OperationCode/operationcode_frontend/blob/master/src/scenes/home/chapterLeader/chapterLeader.js)
- Refactor as required to match our new standards (linting, using components where possible, etc.)

### PR DESCRIPTION
**Pull request by chrisgalvan**
Fixes #167 

image

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit 040db10ecc7e3a917fb30818a01c4d107d592d62

https://deploy-preview-223--operation-code-storybook.netlify.com

File: pages/chapter_leader.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+const ChapterLeader = () => (
+  <>
+    <Head title="Chapter Leaders" />
+
+    <Section title="Chapter Leaders" theme="white">
+      <div>
+        <p>
+          Operation Code is looking for volunteer Chapter Leaders to build local communities
+          nationwide! Tell us more about yourself
+          <a
```
Comment (kylemh) [inline pages/chapter_leader.js]: Can you use the OutboundLink component here?

File: pages/chapter_leader.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            experience with software development.
+          </li>
+        </ul>
+        <h5>Responsibilities</h5>
+        <ul>
+          <li>Enforce the Code of Conduct on Operation Code web communities.</li>
+          <li>Host events, including trainings, talks, hack nights, etc.</li>
+          <li>Build partnerships in the local community.</li>
+          <li>
+            <a
```
Comment (kylemh) [inline pages/chapter_leader.js]: and here.

File: pages/chapter_leader.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            and in-kind donations in support of the mission.
+          </li>
+          <li>Build and integrate the infrastructure necessary to sustain the chapter.</li>
+          <li>Reach out to potential new members, receiving and integrating them to the team.</li>
+          <li>Advocate for and promote the organization in the local community.</li>
+          <li>Help members learn, grow, and find jobs!</li>
+        </ul>
+        <p>
+          Think you are interested in becoming a Chapter Leader? Click
+          <a
```
Comment (kylemh) [inline pages/chapter_leader.js]: and here.

**Comment by chrisgalvan** [comment]
@kylemh thanks for your feedback, I have made the changes

**Pull request merged**